velox/functions/prestosql/window/tests/LeadLagTest.cpp

```diff
@@ -75,6 +81,13 @@ TEST_P(LeadLagTest, offset) {
   // Constant offset.
   assertResults(fn("c0, 2"));
 
+  // Large offset.
+  assertResults(fn("c0, c3"));
+
+  // Large + CONSTANT offset.
+  // 8589934590 == (int64_t)std::numeric_limits<int32_t>::max() * 2
```

Comment:
Turn this comment into code. Like in other places in this PR.

Reply:
Oops, was thinking that it need to literal constant here, will change it.